epan/dissectors/packet-ippusb.c

```diff
@@ -29,18 +29,20 @@
 #define HTTP 0
 #define IPP 1
 
+/* As also defined in IPP dissector */
+#define PRINT_JOB              0x0002
+#define SEND_DOCUMENT          0x0006
+
 #define TAG_END_OF_ATTRIBUTES 0x03
+#define NEWLINE 0x0a
 
-#define CHUNK_LENGTH_MAX 8
+#define CHUNK_LENGTH_MIN 5
 
 #define BITS_PER_BYTE 8
 
-#define LENGTH_OF_RETURN_NEWLINE 2
-
-#define HTTP_IPP_SPLIT_NOT_SET -1
-
-static const guint8 CHUNKED_END[] = { 0x0d, 0x0a };
-static tvbuff_t *chunked_end_tvb = NULL;
+static const guint8 CHUNKED_END[] = { 0x30, 0x0d, 0x0a, 0x0d, 0x0a };
```

Comment:
>0x30 [](start = 38, length = 4)

0x30 or 0x03?  I thought the latter for end of attributes

Reply:
The end of the chunking has a separate value

---
In reply to: [459815523](https://github.com/tjamhar/wireshark/pull/3#discussion_r459815523) [](ancestors = 459815523)